java/src/lib.rs

```diff
@@ -105,7 +123,19 @@ pub extern "system" fn Java_glide_ffi_resolvers_RedisValueResolver_valueFromPoin
     pointer: jlong,
 ) -> JObject<'local> {
     let value = unsafe { Box::from_raw(pointer as *mut Value) };
-    redis_value_to_java(&mut env, *value)
+    redis_value_to_java(&mut env, *value, true)
+}
+
+#[no_mangle]
+pub extern "system" fn Java_glide_ffi_resolvers_RedisValueResolver_valueFromPointerBinary<
```

Comment:
Don't you want to add a second argument to native method `valueFromPointer`?

Reply:
we could, I am not that familiar with the native interface you wrote there, this was the fastest way to production.
Feel free to change this in next PR